### DISCUSSION TITLE
PrimgNg splitbutton component menubutton disabled state

### DISCUSSION THREAD

**Discussion by pouyababaie** [primefaces/community#992] [PrimeNG] [answered]
i wanted to disabled just the menubutton , but there was no option , so i added it myself. i create a pull request for it.

![image](https://github.com/primefaces/community/assets/21077272/5576725f-6a7e-4ffe-9435-180f18891212)

**Comment by cetincakiroglu** [comment]
If they are the new disabled state, they're wrong. The dropdown should also look disabled.

**Reply by pouyababaie** [reply]
I think there is no need for a dropdown menu to be disabled, if the carret is already disabled, how can you even open the dropdown ?

**Reply by cetincakiroglu** [reply, marked as answer]
Hi,

If a button is disabled, it won't allow any interaction related to it. Therefore, the dropdown shouldn't open either. The mentioned visual is not correct in terms of UX, and the PrimeNG core team will not implement such an application. If you want to use it, you can implement it in your own application as a workaround or custom feature when we release passthrough implementation. 

Thank you for your effort and time.
